### DISCUSSION TITLE
Table 20, Panel 2

### DISCUSSION THREAD

**Discussion by pibion** [cssi-pi-community/2022-meeting#56] [Panel 2 roundtable (Monday afternoon)]
Summary for Table 20, panel 2

**Comment by pibion** [comment]
Everybody at the table feels like the postdoc mentoring plan and the data management plans are treated like, "check the boxes".

For CSSI proposals, maybe additional training for the panelists for the data management plan could be helpful?

**Comment by pibion** [comment]
Comment from Amy: can we make a national library for data storage?  Maybe we can make it a movie starring Nick Cage and use the proceeds?

**Comment by RafalGSU** [comment]
I think having meta-repository of all nsf-sponsored data intensive projects (with references to these data sets on nsf site) could  encourage data providers to maintain their data for longer periods.

**Reply by RafalGSU** [reply]
See all VxO repositories that NASA is maintaining/supporting. Virtual Wave Observatory,  Virtual Heliospheric Observatory (VHO), etc.

**Comment by pibion** [comment]
How is CERN/Zenodo guaranteeing 50 years of data??

**Reply by pibion** [reply]
This really feels like an effort that needs help from the top down - individuals alone can't make the centers we need.

**Comment by nancyide** [comment]
How does the ability to store data for long (or long-ish) periods of time affect the possibilities for replication/reproducibility of results?

**Comment by pibion** [comment]
Funding for reproduction of experiments would be really useful.  Some fields have this built in to some extent (there are several dark matter experiments, for example).  But not all.

**Comment by pibion** [comment]
We all feel like we have a strong need for a data facility - that feels like a DOE thing.

Unfortunately, people need to access the data, and that's something the NSF does (but not DOE).

**Comment by pibion** [comment]
At our table, for our archival needs: two people have NASA, one person has NIH (for genomics).

Amy (dark matter/neutrino/nuclear physics) doesn't have an agency with long-term archives.

**Comment by pibion** [comment]
Amy's proposal is: give a few fields a petabyte of storage that's guaranteed for fifty years, or take proposals from fields for it as a contest.

If we know we have storage available, I think my field would figure out how to use it in sensible ways.  We have a lot of hard choices to make, a lot of metadata systems to discuss and build - but none of that will happen right now because there's no archival, accessible facility available.

I think it will be hard to get people to engage with the question of "how much meaningful data do you need, for what purpose" until we know it's a useful discussion.

**Reply by pibion** [reply]
To clarify, I mean a fixed amount of storage!  "You get a petabyte, absolutely no more.  Figure it out."
